### ISSUE TITLE
[Bug] new featurengineering doesn't work with RClassifier models

### ISSUE THREAD

**Issue by egillax** [OHDSI/PatientLevelPrediction#556]
When I added the new featurengineering methods in the latest release I only tested with Cyclops models. Now when running with `XGBoost` I'm getting an error when predicting for the test set. Inside the FE methods for test I'm usually creating a new object for the output but forgot to put the right class of `plpData` so it takes the right path in the xgboost predict method.

**Comment by egillax** [comment]
Some quick grepping reveals this is an issue with `xgboost` , `lightgbm` and `sklearn` models.

**Comment by egillax** [comment]
Fixed in #557
